server/core/enums/wavelengthunits.py

```diff
@@ -0,0 +1,17 @@
+from enum import IntEnum
+
+class WavelengthUnits(IntEnum):
```

Comment:
Astropy has a pretty extensive units package developed. Does it help us to use that?